src/demos/demos.css

```diff
@@ -1,26 +1,33 @@
 body {
   margin: 0;
-  background-color: #dfdfdf;
+  /* background-color: #dfdfdf; */
+  background-color: var(--calcite-app-background-content);
+}
 
+main {
+  margin: 0;
+  padding: 0;
 }
+
 .example-container {
-  padding: 0 24px 24px;
-}
-@media (min-width: 1025px) {
-  .example-container {
-    width: 900px;
-    max-height: 80vh;
-    margin: 12px auto;
-    background-color: var(--calcite-app-background-content);
-    border: 1px solid var(--calcite-app-foreground-hover);
-    box-shadow: 4px 4px 2px var(--calcite-app-foreground-hover);
-    resize: horizontal;
-    overflow-x: scroll;
-  }
+  color: var(--calcite-app-foreground-subtle);
+  max-width: 1024px;
+  min-width: 480px;
+  width: 70vw;
+  padding: var(--calcite-app-cap-spacing-half) var(--calcite-app-side-spacing-double) var(--calcite-app-cap-spacing-double);
+  margin: 0 auto var(--calcite-app-cap-spacing-double);
+  background-color: var(--calcite-app-background-content);
+  box-shadow: var(--calcite-app-shadow-1);
```

Comment:
Can we darken this just a bit. It's very faint.

Reply:
Yo! I nuked the desktop nav on purpose. It mostly just caused a horizontal scroll.

re: darkening 
I actually made the bg color the same and just added a bit of box shadow. If anything, we can prolly just nix the box-shadow and not indicate the center column at all.

Reply:
I went ahead and nixed the box-shadow. 
So clean...
![kleen](https://upload.wikimedia.org/wikipedia/en/7/73/Mr._Clean_logo.png)

Reply:
![image](https://user-images.githubusercontent.com/12503298/69390254-7ae07180-0c83-11ea-9834-d6469806fe44.png)

Reply:
> Yo! I nuked the desktop nav on purpose. It mostly just caused a horizontal scroll.

Agreed. We outgrew it anyway.

Reply:
Can the nav button be positioned as sticky? So its always on the top left even when scrolled down?

Reply:
It can, but it causes other issues with accessing nav items that fall below the scroll. I did some work to have the menu items be in a 3-up grid, but it was taking more time to make right than it's probably worth.

Reply:
Especially with the advanced demos child menu.

Reply:
OK...I take it back. 
Have a look now. @driskull Sticky!

Reply:
Looks good.

The full window demos need the body margin and padding removed.

can we put a title in that navbar header "Calcite App Components"?

Also, the information architecture of the nav items is odd. Can we move the children of the advanced demos under each component they belong to and get rid of the advanced demos.

Reply:
re: title. I'm cool with that. Will add.

re: margin on full width demos
Sorry...that was an oversight on my part.

re: advanced demos.
I like that idea. Can we consider it a later enhancement?

Reply:
yes, later enhancement is fine.